app/drivers/zephyr/dts/bindings/kscan/zmk,kscan-gpio-matrix.yaml

```diff
@@ -15,8 +15,22 @@ properties:
     type: phandle-array
     required: true
   debounce-period:
+    type: int
+    required: false
+    deprecated: true
+    description: Deprecated. Use debounce-on-ms and debounce-off-ms instead.
```

Comment:
```suggestion
    description: Deprecated. Use debounce-down-ms and debounce-up-ms instead.
```

Reply:
Nice catch. I renamed those properties but forgot to update here.